lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/AzureBlockBlobFileSystem.cs

```diff
@@ -123,6 +123,18 @@ public void CreateDirectory(Uri directoryUri)
         {
         }
 
+        /// <summary>
+        /// Checks if uri is a directory uri.
+        /// </summary>
+        /// <param name="uri">uri of the directory/file</param>
+        /// <returns>true if uri is for a directory else false</returns>
+        public bool IsDirectory(Uri uri)
+        {
+            var path = uri.AbsolutePath.Trim('/');
+            var blobItems = _client.ListBlobsSegmented(path, false, BlobListingDetails.Metadata, 1, null, null, null).Results;
```

Comment:
>1 [](start = 97, length = 1)

super-nit: Make a static variable. It helps self document the code.

Reply:
creating a local variable

---
In reply to: [194554180](https://github.com/apache/reef/pull/1473#discussion_r194554180) [](ancestors = 194554180)

Reply:
const static?

---
In reply to: [194580367](https://github.com/apache/reef/pull/1473#discussion_r194580367) [](ancestors = 194580367,194554180)

Reply:
static wont apply to local variable. Not sure if you wanted to make this a class variable. I have used const only

---
In reply to: [195218006](https://github.com/apache/reef/pull/1473#discussion_r195218006) [](ancestors = 195218006,194580367,194554180)